README.md

```diff
@@ -31,3 +29,7 @@ For examples:
 ## Create DataBase
 ---
 Run this command in your terminal to create database: `psql -f commands.sql`
+
+## Run server
+---
+Run server: `rackup`
```

Comment:
Also add instruction what to do with that next: `open <URL>` - put the correct URL
(`open` is a real linux command, check it)